net/nimble/controller/src/ble_ll_scan.c

```diff
@@ -201,7 +202,11 @@ ble_ll_aux_scan_cb(struct ble_ll_sched_item *sch)
     }
 
     STATS_INC(ble_ll_stats, aux_fired_for_read);
-    ble_phy_wfr_enable(BLE_PHY_WFR_ENABLE_RX, 0, BLE_LL_SCHED_ADV_MAX_USECS);
+
+    wfr_usec = scansm->cur_aux_data->offset_units ? 300 : 30;
+    /* Lets add one tick to make sure we will not miss aux */
+    wfr_usec += os_cputime_ticks_to_usecs(1);
```

Comment:
this is hack and it should be marked appropriately

Reply:
actually we don't need it anymore as ble_ll_sched_aux_scan() is fixed and takes rem_usec into account when scheduling. I will remove +1